include/aikido/planner/dart/ConfigurationToTSR.hpp

```diff
@@ -24,7 +24,11 @@ class ConfigurationToTSR : public Problem
   /// current state of when called.
   /// \param[in] endEffectorBodyNode BodyNode to be planned to move to a desired
   /// TSR.
-  /// \param[in] maxSamples Maximum number of TSR samples to plan to.
+  /// \param[in] maxSamplingTries Maximum number of times to try when
+  /// sampling from the TSR.
+  /// \param[in] batchSize Number of TSR samples to include per batch.
```

Comment:
I would consider clarifying that this is a `ranked batch`. Batching itself doesn't add any additional functionality (since we already do `maxSamplingTries` underlying samples) except in that we carry out ranking within a batch.